x-pack/test/functional/apps/machine_learning/anomaly_detection/categorization_job.ts

```diff
@@ -0,0 +1,392 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import expect from '@kbn/expect';
+
+import { FtrProviderContext } from '../../../ftr_provider_context';
+import { CATEGORY_EXAMPLES_VALIDATION_STATUS } from '../../../../../legacy/plugins/ml/common/constants/new_job';
+
+// eslint-disable-next-line import/no-default-export
+export default function({ getService }: FtrProviderContext) {
+  const esArchiver = getService('esArchiver');
+  const ml = getService('ml');
+
+  const jobId = `categorization_${Date.now()}`;
+  const jobIdClone = `${jobId}_clone`;
+  const jobDescription =
+    'Create categorization job based on the categorization_functional_test dataset with a count rare';
+  const jobGroups = ['automated', 'categorization'];
+  const jobGroupsClone = [...jobGroups, 'clone'];
+  const detectorTypeIdentifier = 'Rare';
+  const categorizationFieldIdentifier = 'field1';
+  const categorizationExampleCount = 5;
+  const bucketSpan = '15m';
+  const memoryLimit = '15mb';
+
+  function getExpectedRow(expectedJobId: string, expectedJobGroups: string[]) {
+    return {
+      id: expectedJobId,
+      description: jobDescription,
+      jobGroups: [...new Set(expectedJobGroups)].sort(),
+      recordCount: '2,399',
+      memoryStatus: 'ok',
+      jobState: 'closed',
+      datafeedState: 'stopped',
+      latestTimestamp: '2016-02-11 23:56:59',
+    };
+  }
+
+  function getExpectedCounts(expectedJobId: string) {
+    return {
+      job_id: expectedJobId,
+      processed_record_count: '1,501',
+      processed_field_count: '4,798',
+      input_bytes: '180.6 KB',
+      input_field_count: '4,798',
+      invalid_date_count: '0',
+      missing_field_count: '0',
+      out_of_order_timestamp_count: '0',
+      empty_bucket_count: '0',
+      sparse_bucket_count: '0',
+      bucket_count: '239',
+      earliest_record_timestamp: '2016-02-07 00:02:50',
+      latest_record_timestamp: '2019-11-21 06:01:13',
+      input_record_count: '2,399',
+      latest_bucket_timestamp: '2016-02-11 23:30:00',
+    };
+  }
+
+  function getExpectedModelSizeStats(expectedJobId: string) {
+    return {
+      job_id: expectedJobId,
+      result_type: 'model_size_stats',
+      model_bytes_exceeded: '0.0 B',
+      model_bytes_memory_limit: '15.0 MB',
+      total_by_field_count: '3',
+      total_over_field_count: '0',
+      total_partition_field_count: '2',
+      bucket_allocation_failures_count: '0',
+      memory_status: 'ok',
+      timestamp: '2016-02-11 23:00:00',
```

Comment:
This needs an update to match the actual data (should be `2019-11-21 05:45:00`).